### PR TITLE
Last doc reading

### PR DESCRIPTION
**Pull request by nchampollion**
Closes https://github.com/OGGM/oggm/issues/###

- [ ] Tests added/passed
- [ ] Fully documented
- [ ] Entry in `whats-new.rst`

### CODE REVIEW

**Comment by fmaussion** [comment]
Your PR is not very clean again :-) In the future please try to start from a fresh upstream/master state, ideally on a local branch different form master. I'm going to assume that the new changes begin at 77f704c ?

**Comment by nchampollion** [comment]
Yes, I know. Sorry. I wasn't able to start from a totally fresh upstream/master. I tried either to rebase completely my master branch or to create a clean new branch from upstream/master without success, all the times the 2 commits about 'multi-processing' remains, I did not figure out why ... I hope that with this PR, my master branch will be totally clean.

Yes, the new changes start at 77f704c !

**Comment by fmaussion** [comment]
> I hope that with this PR, my master branch will be totally clean.

no it doesn't work this way. A few tips:

- after this is merged, hard reset your local master:

```
$ git fetch upstream
$ git checkout master
$ git reset --hard upstream/master
```

- after that, **stop adding commits to local master**. Local master will always be a way to update from upstream master, with:

```
$ git fetch upstream
$ git checkout master
$ git merge upstream/master
```

- when working on a PR, **create a new branch from local master** (which is up to date with upstream/master):

```
# update local master
$ git fetch upstream
$ git checkout master
$ git merge upstream/master
# make new branch
$ git checkout -b my-feature
.... work + commit(s)...
$ git push origin my-feature
... open PR on OGGM
```
- once the PR is merged, your `my-feature` branch is useless/messy (like your master now). you can delete it, (or hard reset it to restart from scratch, which can be dangerous)

**Comment by nchampollion** [comment]
Ahhh, I think I guess my mistake.

I used:

`$ git fetch upstream`
`$ git checkout master`
`$ git rebase upstream/master`

Instead of:

`$ git fetch upstream`
`$ git checkout master`
`$ git reset --hard upstream/master`

Thanks for all the tips, following them in the future. I would like to add to the previous tips that it seems easier to work locally on our own laptop than directly online in github.

**Comment by fmaussion** [comment]
>  I would like to add to the previous tips that it seems easier to work locally on our own laptop than directly online in github.

Yes. I never heard anything good from the git GUIs.

**Comment by fmaussion** [comment]
and always remember that

`$ git reset --hard upstream/master`

Is going to overwrite every local change you might have, so to use with care...

**Comment by fmaussion** [comment]
https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular ;-)

File: docs/input-data.rst [hunk 1]
```diff
@@ -59,30 +59,35 @@ Currently, there are six available levels of pre-processing:
...
   the processing chain can be re-run from any stage in them.
 - **Level 4**: same as level 3 but with all intermediate output files removed.
   The strong advantage of level 4 files is that their size is considerably
   reduced, at the cost that certain operations (like plotting on maps or
   running the bed inversion algorithm again) are not possible anymore.
 - **Level 5**: on top of level 4, an additional historical simulation is run
   from the RGI date to the last possible date of the baseline climate file
-  (for example end of 2019 for CRU and ERA5).
-  The state of the glacier as month 01, (hydrological) year 2020 can then be
+  (currently december 2019 for CRU and ERA5).
```
Comment (fmaussion) [inline docs/input-data.rst:71]:
```suggestion
  (currently January 1st 2020 at 00H for CRU and ERA5).
```

File: docs/input-data.rst [hunk 2]
```diff
@@ -157,7 +162,7 @@ for your research question, and to start your runs from level 5 if possible.
   ``dl_cache_dir`` folder (see :ref:`system-settings`).
 
 
-Available pre-processed configurations
+Available pre-processed configurations:
```
Comment (fmaussion) [inline docs/input-data.rst:165]:
```suggestion
Available pre-processed configurations
```

File: docs/practicalities.rst [hunk 3]
```diff
@@ -53,7 +53,9 @@ The default in OGGM is:
 
 ``-1`` means that all available processors will be used.
 
-The following environment variables will override these settings:
+The following environment variables will override these settings (see
+`_https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-environments.html`_
```
Comment (fmaussion) [inline docs/practicalities.rst:57]: That's not what's meant here! Will change in a separate PR.

**Comment by fmaussion** [comment]
Thanks!

**Comment by nchampollion** [comment]
> https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular ;-)

Thanks, I'm already outdated since I'm writing like many years ago ;-)

**Pull request merged**